### PR TITLE
feat: relative sizing

### PR DESCRIPTION
**Pull request by eirikbacker**
Work on #2508 
Minor adjustments will be done in #2665

### CODE REVIEW

File: packages/css/pagination.css
Comment (Barsnes) [inline packages/css/pagination.css:1]: This does not change size

File: packages/css/input.css
Comment (Barsnes) [inline packages/css/input.css:1]: The circle on a selected radio is too small on radio? 
image

Should discuss XS size on form components:
image

File: packages/react/src/types.ts [hunk 1]
```diff
@@ -1,3 +1,5 @@
+export type Size = 'xs' | 'sm' | 'md' | 'lg';
```
Comment (Barsnes) [inline packages/react/src/types.ts:1]: Discussed this on slack via chat

**Review by Barsnes — COMMENTED** [review]
Mostly looks good, found a few things.
For info:
We decided via slack chat to **not** include `xs` in the shared type, as there are components we don't want this for

File: packages/css/error-summary.css
[the remark this answers is not in the document]
Comment (Barsnes) [reply packages/css/error-summary.css:1]: Disregard this, I see this has been fixed.

File: packages/css/chip.css
[the remark this answers is not in the document]
Comment (eirikbacker) [reply packages/css/chip.css:1]: yes, chip will be fixed in #2669 ☺️

File: packages/css/input.css
Comment (eirikbacker) [reply packages/css/input.css:1]: Yes, this is postponed to #2665 as design needs to verify behaviour

File: packages/css/pagination.css
Comment (eirikbacker) [reply packages/css/pagination.css:1]: Fixed :)

**Comment by mimarz** [comment]
Suggest renaming our component prop `size` to be just `data-size`. During testing I found it a bit unintuitive to use `data-size` on global level and `size` on ~some~ components.

**Comment by Barsnes** [comment]
> Suggest renaming our component prop `size` to be just `data-size`. During testing I found it a bit unintuitive to use `data-size` on global level and `size` on ~some~ components.
> 
> #2671

Would this not be unintuitive for react users? We should probably add a global type for this if we remove it

**Review by mimarz — APPROVED** [review]
LGTM! 
size-is-everything-neil-degrasse-tyson

**Pull request merged**